keyboards/crkbd/keymaps/soundmonster/config.h

```diff
@@ -51,11 +51,17 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
     #define RGBLIGHT_VAL_STEP 17
 #endif
 
-#define OLED_FONT_H "keyboards/crkbd/keymaps/soundmonster/glcdfont.c"
-// #define OLED_FONT_WIDTH 5
-// #define OLED_FONT_HEIGHT 7
+
+#ifdef OLED_ENABLE
+#   define SPLIT_LAYER_STATE_ENABLE
+#   define SPLIT_LED_STATE_ENABLE
+#   define SPLIT_MODS_ENABLE
+#   define SPLIT_OLED_ENABLE
+#   define OLED_FONT_H "keyboards/crkbd/keymaps/soundmonster/glcdfont.c"
+#endif
 
 #ifdef RGB_MATRIX_ENABLE
+#   define SPLIT_TRANSPORT_MIRROR // needs key presses from master on the slave side
```

Comment:
```suggestion

```
This isn't required as it's already set in the keyboard level config for both rev1 and r2g.
https://github.com/qmk/qmk_firmware/blob/1b953ac1e2a0b6f4bb91dea71339d1ccfc25f523/keyboards/crkbd/rev1/config.h#L39